src/test/java/com/impactupgrade/nucleus/it/MailchimpCommunicationServiceIT.java

```diff
@@ -125,21 +121,20 @@ public void syncUnsubscribes() throws Exception {
     assertEmailsStatus(cleanEmails, "cleaned", listId, mailchimpClient);
 
     // Sync unsubscribes MC >> SF
-    for (CommunicationService communicationService : env.allCommunicationServices()) {
-      if (communicationService instanceof MailchimpCommunicationService) {
-        communicationService.syncUnsubscribes(beforeBulkImport);
-      }
-    }
+    env.communicationService("mailchimp").syncContacts(beforeBulkImport);
 
     crmContacts = env.primaryCrmService().getContactsByEmails(unsubscribeEmails);
+    assertTrue(!crmContacts.isEmpty());
```

Comment:
@VSydor SUPER picky: assertFalse(crmContacts.isEmpty()); A little easier to read?